catalog/ui/catalog-ui-search/src/main/webapp/component/input/autocomplete/input-autocomplete.less

```diff
@@ -0,0 +1,35 @@
+@{customElementNamespace}input.is-autocomplete {
+    .select2-container--default .select2-selection--single {
+        background: @darken-inherited-background-color;
+        color: @button-text-color;
+        border: 1px solid @heavily-lighten-inherited-background-color;
+        border-radius: @border-radius;
+        height: auto;
+
+        .select2-selection__rendered {
+            line-height: @minimumButtonSize;
+            padding-right: @minimumButtonSize;
+        }
+
+        .select2-selection__arrow {
+            top: 0px;
+            height: 100%;
+            width: @minimumButtonSize;
+        }        
+    }
+
+
+}
+
+// can't namespace because gets appended to doc
+.select2-dropdown {
+    z-index: @zIndexDropdown;
+    background-color: @dropdown-background-color;
+    border-radius: @border-radius;
+    border-color: @heavily-lighten-inherited-background-color;
+    color: @button-text-color;
+}
+
+.select2-container--default .select2-results__option--highlighted[aria-selected] {
+    background: @primary-color;
```

Comment:
We can use `@lighten-inherited-background-color` here to bring it in line with the other dropdowns.